conf/seeds.json

```diff
@@ -0,0 +1,10 @@
+{
+  "firstName": "Simon",
+  "lastName": "Belmont",
+  "username": "admin",
+  "email": "simon@bellmont.com",
+  "bio": "vampires beware",
+  "admin": true,
+  "salt": "33a551c898d283c48e121a3b629b23dc785c56837b3ef51cc6702a8506317eda",
```

Comment:
I'm not sure about including the salt and hash in the seed file. Maybe we should consider using a hashing library like BCrypt to create salts for us.